diff --git a/include/acpi/acpixf.h b/include/acpi/acpixf.h
index 459d6981ca96..4a18a9016665 100644
--- a/include/acpi/acpixf.h
+++ b/include/acpi/acpixf.h
@@ -228,7 +228,7 @@ ACPI_INIT_GLOBAL(u32, acpi_gbl_max_loop_iterations, ACPI_MAX_LOOP_TIMEOUT);
  * use such a package, the unresolved element(s) will be replaced with NULL
  * elements.
  */
-ACPI_INIT_GLOBAL(u8, acpi_gbl_ignore_package_resolution_errors, FALSE);
+ACPI_INIT_GLOBAL(u8, acpi_gbl_ignore_package_resolution_errors, TRUE);
 
 /*
  * This mechanism is used to trace a specified AML method. The method is
